src/client/pages/Profile.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+import useFetch from '../hooks/useFetch';
+
+import Button from '../components/Button';
+
+import '../style/Profile.css';
+
+function Profile() {
+  const location = useLocation();
+  const history = useHistory();
+  const [username, setUsername] = useState();
+  const [email, setEmail] = useState();
+  const [about, setAbout] = useState();
+  const [name, setName] = useState();
+  const [loc, setLocation] = useState();
+  const [verified, setVerified] = useState();
+  const [DOB, setDOB] = useState();
+  const [ID, setID] = useState(2);
+
+  const endPoint = `/api/getProfile/${ID}`;
+  const [isLoading, data, error] = useFetch(endPoint);
+  const dataList = data || [];
+
+
+  useEffect(() => {
+    setUsername(dataList.Username);
+    setEmail(dataList.Email);
+    setAbout(dataList.About);
+    setName(dataList.FirstName);
+    setLocation(dataList.Locat);
+    setDOB(dataList.DOB);
+    setVerified(dataList.Type);
+  }, [dataList.About, dataList.DOB, dataList.Email, dataList.FirstName, dataList.Locat,
+    dataList.Type, dataList.Username]);
+
+  let verifiedStatus;
+  if (verified === 1) {
+    verifiedStatus = <h3 className="profile-verified-yes"> &#x2714; Verified</h3>;
+  } else {
+    verifiedStatus = <h3 className="profile-verified-no">  &#10007; Not Verified</h3>;
+  }
+
+  const handleUpdate = (event) => {
+    const updatePoint = `/api/updateProfile/${ID}`;
+    event.preventDefault();
+    const profileData = {
+      email,
+      username,
+      about,
+      name,
+      loc,
+      DOB,
+      ID
+    };
+    const request = {
+      method: 'PUT',
+      mode: 'cors',
+      headers: {
+        'Content-Type': 'application/json',
+        'Content-Length': profileData.toString().length.toString()
+      },
+      body: JSON.stringify(profileData)
+    };
+    fetch(updatePoint, request);
```

Comment:
If it is an asynchronous call here, don't forget to handle callback.

Reply:
Ok. Get API is still work in progress so I will incorporate the comment in next update